app/controllers/campaigns_controller.rb

```diff
@@ -5,12 +5,7 @@ class CampaignsController < ApplicationController
 
   # GET /campaigns
   def index
-    @campaigns = if params[:active].present?
```

Comment:
these changes were from alice's PR but I'm not sure why they showed up in my files changed? :/